src/snovault/elasticsearch/create_mapping.py

```diff
@@ -88,11 +84,16 @@ def schema_mapping(name, schema):
             mapping = schema_mapping(k, v)
             if mapping is not None:
                 properties[k] = mapping
-        return {
-            'type': 'object',
-            'include_in_all': False,
-            'properties': properties,
```

Comment:
If this always had `properties` why the if/else now?

Reply:
if there is no mapping directive exists in the schema, object should be stored without its fields being analyzed. `else` clause prevents unnecessary dynamic mapping and `update_mapping` events in elasticsearch.

Reply:
This doesn't seem to be working properly for some fields, e.g. `cloud_metadata` in files. (Might have something to do with it being a calculated_property.)

<img width="306" alt="Screen Shot 2020-10-27 at 12 18 01 PM" src="https://user-images.githubusercontent.com/16053922/97353342-2c533400-1851-11eb-8123-f02dc182da8f.png">

<img width="187" alt="Screen Shot 2020-10-27 at 12 18 19 PM" src="https://user-images.githubusercontent.com/16053922/97353348-2f4e2480-1851-11eb-8c10-182696a893df.png">

Before and after. Now you can't look up cloud_metadata.url=* for example.

Reply:
Right, previous assumption was that if the field was important, it would be in the schema. This can be fixed, implication is that any other object inserted that doesn't have a mapping directive in the schema will have all of its field values be analyzed as keyword.